tests/lora/test_baichuan.py

```diff
@@ -81,13 +81,16 @@ def test_baichuan_tensor_parallel_equality(baichuan_lora_files):
     del llm_tp1
     cleanup()
 
-    llm_tp2 = vllm.LLM(MODEL_PATH,
-                       enable_lora=True,
-                       max_num_seqs=16,
-                       max_loras=4,
-                       max_lora_rank=64,
-                       tensor_parallel_size=2,
-                       trust_remote_code=True)
+    llm_tp2 = vllm.LLM(
+        MODEL_PATH,
+        enable_lora=True,
+        max_num_seqs=16,
+        max_loras=4,
+        max_lora_rank=64,
+        tensor_parallel_size=2,
+        trust_remote_code=True,
+        fully_sharded_loras=True,
```

Comment:
can we parametrize this and test both fully sharded True and False?

Reply:
Done， please review again, thank you.